src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -112,6 +112,18 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):#pylint:
 register_cli_argument('vm extension image', 'type', CliArgumentType(options_list=('--name', '-n')))
 register_cli_argument('vm extension image', 'latest', CliArgumentType(action='store_true'))
 
+register_cli_argument('vmss extension', 'extension_name', name_arg_type, id_part='name', help='Name of extension')
+register_cli_argument('vmss extension', 'vmss_name', help='Scale set name')
+register_cli_argument('vmss extension', 'auto_upgrade_minor_version', CliArgumentType(action='store_true'))
```

Comment:
We should definitely get rid of it, as we have third parties submitting code with that anti-pattern based on the fact that our packages have it. But, the issue should be sufficient for now.

---
In reply to: [76143912](https://github.com/Azure/azure-cli/pull/737#discussion_r76143912) [](ancestors = 76143912,76134377,76100584,76077802)